assets/package.json

```diff
@@ -1,11 +1,18 @@
 {
   "repository": {},
   "description": " ",
-  "license": "MIT",
+  "license": "agpl-3.0",
```

Comment:
Interesting that you changed the license hehe
I'm curious

Reply:
The license didn’t actually change - Keila has always been AGPLv3 from the start, but the package.json originally created by the Phoenix generator didn’t reflect that.